Connectors/Rhino/Speckle.Connectors.GrasshopperShared/Components/Collections/CreateCollection.cs

```diff
@@ -64,6 +64,13 @@ protected override void SolveInstance(IGH_DataAccess dataAccess)
       }
     }
 
+    // validate for duplicate application IDs across the entire collection hierarchy
+    if (HasDuplicateApplicationIds(rootCollection))
```

Comment:
negligible impact on solve times (tested with loading big revit model and creating collection from received model)